src/scripts/components/stories/pilot-story/components/globe/globe.tsx

```diff
@@ -63,6 +65,16 @@ const Globe: FunctionComponent<Props> = ({
     }
   }, [containerRef, globe]);
 
+  useEffect(() => {
+    if (containerRef.current) {
+      containerRef.current.style.top = `${initialPosition.top}%`;
+      containerRef.current.style.left = `${initialPosition.left}%`;
+      containerRef.current.style.right = `${initialPosition.right}%`;
+      containerRef.current.style.bottom = `${initialPosition.bottom}%`;
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
You should be able to memoize `initialPosition` and get rid of the comment here

Reply:
Can you please explain how?

Reply:
As `initialPosition` is passed down as a prop, you could define it in the parent like 

```
  const memoizedRelativePosition = useMemo(
    () => ({top: 0, left: -100, right: 0, bottom: 0}),
    []
  );
```

Reply:
Ok, will be removed with next PR.